packages/cms-cli/bin/cli.js

```diff
@@ -22,7 +22,7 @@ const fetchCommand = require('../commands/fetch');
 const filemanagerCommand = require('../commands/filemanager');
 const secretsCommand = require('../commands/secrets');
 
-const SCRIPT_NAME = 'banjo';
+const SCRIPT_NAME = 'hs';
```

Comment:
Can we assign this dynamically somehow based on the command?

Reply:
It's a tad hacky, but not too bad.  We can change the `bin` in the package.json to pass a flag

```
  "bin": {
    "banjo": "./bin/hs.js --command=banjo",
    "hs": "./bin/hs.js --command=hs",
    "hscms": "./bin/hs.js --command=hscms"
  },
````

and then parse it out and assign the script name to it

```
const commandName = process.argv.find(a => a.includes('--command='));
const SCRIPT_NAME = commandName
  ? commandName.split('--command=')[1] || 'hs'
  : 'hs';
```

Testing locally it worked

Reply:
Actually.  `npm_lifecycle_event` is a thing.  Not really seeing much documentation on it, but I think it would work

Reply:
What if we just use the default for `yargs` and don't call `yargs.scriptName()`? I think it works as expected.

Reply:
![image](https://user-images.githubusercontent.com/5521743/92134126-24d65800-edd7-11ea-98c2-6a0d8c842a04.png)

That seems to make it use the file name instead.  The `npm_licecycle_event` thing seems to work well tho.  I'll try a couple older versions of node just to be safe

Reply:
Tested with node 8.9.1 and works like a champ

Reply:
Out of curiosity, how were you testing?

Reply:
We may want to create an `hs` file inside `bin` and have it require cli.js. 

```bash
#!/usr/bin/env node

require('./cli');
```

Reply:
Swapped to node 8.9.1 and locally called the script

![2020-09-03 11 55 04](https://user-images.githubusercontent.com/5521743/92138519-79300680-eddc-11ea-86e6-bea7c01cc244.gif)